dhall-to-cabal.cabal

```diff
@@ -4,9 +4,10 @@ cabal-version: 2.0
 --
 -- Do not edit it by hand, because your changes will be over-written!
 --
--- Instead, edit the source Dhall file, namely
--- 'dhall-to-cabal.dhall', and re-generate this file by running
--- 'dhall-to-cabal -- dhall-to-cabal.dhall > dhall-to-cabal.cabal'.
+-- Instead, edit the source Dhall file (which may have the
+-- '.dhall' extension) and re-run dhall-to-cabal, passing the
+-- source file's name as its argument and redirecting output to
+-- 'dhall-to-cabal.cabal' (this file).
```

Comment:
One last thing, could you regenerate this file with the instructions given? That is:

```
dhall-to-cabal -- dhall-to-cabal.dhall > dhall-to-cabal.cabal
```

That will keep this warning the same, and it's slightly more instructive.

Reply:
mmm, i did recreate the file using dhall-to-cabal itself (built with master version). I think it is caused by building the executable in windows, cause win paths doesnt satisfy the [isAcceptableCommentChar](https://github.com/dhall-lang/dhall-to-cabal/blob/master/exe/Main.hs#L206) check.
Moreover the suggestion is only valid in *nix envs (should be `type dhall-to-cabal.dhall | dhall-to-cabal > dhall-to-cabal.cabal` for win)
But whatever, i guess there are not much dhall-to-cabal users in windows but me for now :wink: 
so i will update the message (make it suitable for window would be in another pr)